### ISSUE TITLE
[Bug]: bvt occasionally failed by internal error: force flush failed: internal error: timeout when run select mo_ctl('dn','checkpoint','');

### ISSUE THREAD

**Issue by guguducken** [matrixorigin/matrixone#12965]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):https://github.com/matrixorigin/matrixone/actions/runs/6979471228/job/18993462166
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="1436" alt="image" src="https://github.com/matrixorigin/matrixone/assets/22561920/9c81835d-4049-4fe4-a935-3b1ccf11f485">

**Comment by aptend** [comment]
和 #12645 类似，现已找到一个解决方法，但是没有找到根本原因

**Comment by XuPeng-SH** [comment]
fixed by #13218
